gui/slick/js/new/config_postProcessing.js

```diff
@@ -0,0 +1,2 @@
+$('#config-components').tabs();
```

Comment:
Just checked and I think we still need it but I can move them all to a single file.